inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/hive/HiveSinkDTO.java

```diff
@@ -152,4 +159,13 @@ public static HiveTableInfo getHiveTableInfo(HiveSinkDTO hiveInfo, List<HiveColu
         return tableInfo;
     }
 
+    private HiveSinkDTO decodePassword() {
+        if (this.password != null) {
```

Comment:
Please use the `StringUtils.isNotBlank()` method for null checks, tks.

Reply:
fixed